src/appContent.js

```diff
@@ -14,7 +14,7 @@ const fieldLabels = {
     class: "Klasa"
   },
   applicant: {
-    name1: "Imię  wnioskodawcy",
+    firstName1: "Imię wnioskodawcy",
```

Comment:
That's maybe not on your code, but I plead you on my knees, can you sort that mess. Let be `lastName1` next to `lastName2`. We have `firstName1` and `name2` (don't ask me why?) We have `street1` here and `address1` in fixtrures and templates. Thats why they don't render it's content on form.

Reply:
Fixed in other PR